fdbclient/Tuple.h

```diff
@@ -47,6 +47,7 @@ struct Tuple {
 	Tuple& appendFloat(float);
 	Tuple& appendDouble(double);
 	Tuple& appendNull();
+	Tuple& appendVersionstamp(StringRef const& str);
```

Comment:
The other bindings, rather than using raw bytes for versionstamps, introduced a new type that wrapped the array. This had a few purposes, including that the length check could be included in the Versionstamp struct rather than being in the Tuple code, and it also was where we would later put the data about whether the versionstamp was incomplete or not (as an additional field in the struct). This doesn't support incomplete versionstamps. so it's maybe a little less critical, but adding a type to encapsulate the data may be desired.